README.md

```diff
@@ -1,80 +1,104 @@
 
+## Article V
 
-# Article V
-## Officers (Titles, Term of Office, How and When Elected)
+### Officers (Titles, Term of Office, How and When Elected)
 
-Each officer position with the exception of System Architect serves until the Spring semester election following their election. Nominations for each generally elected position, that is all positions other than System Architect, can be submitted by any member beginning one month before an election and ending at the election of the particular officer. At the last meeting of the Spring semester or the last meeting of the Fall semester, the generally elected positions that are open, that is all positions other than System Architect that are open, will be filled by a majority vote of the nominees by active members. The position of System Architect will be filled by a unanimous vote of the current officers whenever the position is open.
+Each officer position serves until the Spring semester election following their election. Nominations for each generally elected position can be submitted by any member beginning one month before an election and ending at the election of the particular officer. At the last meeting of the Spring semester or the last meeting of the Fall semester, the generally elected positions that are open will be filled by a majority vote of the nominees by active members.
 
 Officer positions are as follows:
+
 * President
 * Vice President
-* System Architect
 * Treasurer
 * Secretary
-* Webmaster
+* Competition and Events Manager
+* Training Manager
+
+Should an officer be unable to fulfill their duties due to health issues, they are relieved of their position, and their position is to be filled at the next meeting by a snap election.
+
+## Article VI
+
+### Appointed Positions (Titles, Term of Office, How Selected)
+
+Each appointed position, with the exception of Lab Administrator, serves at-will of the board of Officers, is appointed by 2/3 majority of Officers' approval, and is relieved via graduation or 2/3 majority of sitting Officers' approval. The position of Lab Administrator will be filled by a unanimous vote of the current officers and advisors, relieved only via graduation or unanimous vote of current officers and advisors.
+
+Appointed positions are as follows:
+
+* Lab Administrator
+* Website Administrator
+* Social Event Manager
+* CyberRange Project Manager
+* Weekly Meeting Manager
+* Blue Team Trainer
+* Red Team Trainer
+
+Should an appointee be unable to fulfill their duties due to health issues, they are relieved of their duties, and their position is to remain vacant until another is appointed by the Officers.
```

Comment:
It should be specified who determines that the appointee is unable to fulfill their duties due to health conditions if this is intended to be an easier way to remove an officer who is in poor health.

Frankly, it could be left up to the removal clause specified on line 48.

Reply:
More of meant to be able to cover if someone unfortunately were to drop dead. Hopefully unlikely, but still something that might need to be accounted for

Reply:
What you are trying to avoid is a case where an officer is removed adversarially by one or more officer who tries to evict an appointee using the health exception as a back door.

Reply:
There should also probably be some notice provision where the appointee is served notification that the office corps intends to vote to remove them allowing them some time to respond.

Reply:
I don't think the scenario where officers attempt to evict someone from a position using the health clause is particularly likely. But perhaps the wording could be changed to "should an appointee cede their position due to health issues their position is to remain vacant until another is appointed by the Officers." Unless this does not capture the implied intent of replacing a member that has become incompetent in their duties somehow.

I don't think the CU Cyber constitution need be treated as a contract for a major corporation where millions of dollars are at stake; hostile takeovers are not a serious concern for a casual student organization in my opinion.

Reply:
I like @AdamVanScyoc ‘s wording. 

I agree it’s not quite as high stakes, but CUCyber does have several thousand in assets, and it doesn’t hurt to express the desired intent clearly when it’s doesn’t cost very much time or effort to do it right.

Reply:
We could probably just shorten it to "cede", as I think that would cover both health issues, and if they just outright quit.

`Ahould an appointee cede their position their position is to remain vacant until another individual is appointed by the Officers.`